Packs/CiscoAMP/Integrations/CiscoAMP/CiscoAMP_test.py

```diff
@@ -0,0 +1,1018 @@
+
+
+def test_computer_user_activity_list_command(requests_mock, mock_client):
+    """
+    Scenario:
+    -   Get user activity on computers.
+    Given:
+    -   The user has entered a username.
+    When:
+    -    cisco-amp-computer-user-activity-get is called.
+    Then:
+    -   Ensure outputs_prefix is correct.
+    -   Ensure isn't in the outputs.
+    """
+    mock_response = load_mock_response('computer_user_activity_response.json')
+    requests_mock.get(
+        f'{BASE_URL}/computers/user_activity',
+        json=mock_response
+    )
+
+    args = {
+        'username': 'johndoe'
+    }
+
+    from CiscoAMP import computer_user_activity_list_command
+    response = computer_user_activity_list_command(mock_client, args)
+
+    assert response.outputs_prefix == 'CiscoAMP.ComputerUserActivity'
+    assert_output_has_no_links(response.outputs)
+
+
+@pytest.mark.parametrize(
+    'args',
+    [({'connector_guid': '1', 'page': '1', 'page_size': '1'})]
+)
+def test_computer_user_trajectory_list_command(requests_mock, mock_client, args):
+    """
+    Scenario:
+    -   Get a computer's trajectory with pagination.
+    Given:
+    -   The user has entered a connector_guid, page and page_size.
+    When:
+    -    cisco-amp-computer-user-trajectory-get is called.
+    Then:
+    -   Ensure outputs_prefix is correct.
+    -   Ensure length of the outputs is correct.
+    -   Ensure connector_guid is in the outputs.
+    """
+    mock_response = load_mock_response('computer_user_trajectory_response.json')
+    requests_mock.get(
+        f'{BASE_URL}/computers/{args["connector_guid"]}/user_trajectory',
+        json=mock_response
+    )
+
+    from CiscoAMP import computer_user_trajectory_list_command
+    response = computer_user_trajectory_list_command(mock_client, args)
+
+    assert response.outputs_prefix == 'CiscoAMP.ComputerUserTrajectory'
+    assert len(response.outputs) == 1
+    assert 'connector_guid' in response.outputs[0]
+
+
+def test_computer_vulnerabilities_list_command(requests_mock, mock_client):
+    """
+    Scenario:
+    -   Get vulnerabilities of a computer.
+    Given:
+    -   The user has entered a connector_guid.
+    When:
+    -    cisco-amp-computer-vulnerabilities-get is called.
+    Then:
+    -   Ensure outputs_prefix is correct.
+    -   Ensure length of the outputs is correct.
+    -   Ensure connector_guid is in the outputs.
+    """
+    args = {
+        'connector_guid': '12345'
+    }
+
+    mock_response = load_mock_response('computer_vulnerabilities_response.json')
+    requests_mock.get(
+        f'{BASE_URL}/computers/{args["connector_guid"]}/vulnerabilities',
+        json=mock_response
+    )
+
+    from CiscoAMP import computer_vulnerabilities_list_command
+    response = computer_vulnerabilities_list_command(mock_client, args)
+
+    assert response.outputs_prefix == 'CiscoAMP.ComputerVulnerability'
+    assert len(response.outputs) == 1
+    assert 'connector_guid' in response.outputs[0]
```

Comment:
please write tests that verify the output values (here, and accross the file)

Reply:
I've added here a function to validate there are no `links` in the output and in another function that was missing it.

Reply:
Unit tests should check sure that the output is as expected, not just the lack of `links` or the length of the output. See the [tests for ipinfo_v2](https://github.com/demisto/content/blob/master/Packs/ipinfo/Integrations/ipinfo_v2/ipinfo_v2_test.py) or [YodaSpeak](https://github.com/demisto/content/blob/master/docs/tutorial-integration/YodaSpeak/Integrations/YodaSpeak/YodaSpeak_test.py) as reference.